addons/web_editor/static/lib/odoo-editor/src/commands/commands.js

```diff
@@ -403,12 +403,12 @@ export function toggleFormat(editor, format) {
     const {anchorNode, anchorOffset, focusNode, focusOffset} = editor.document.getSelection();
     const style = styles[format];
     const selectedTextNodes = getSelectedNodes(editor.editable)
-        .filter(n => n.nodeType === Node.TEXT_NODE && n.nodeValue.trim().length);
-    const isAlreadyFormatted = style.is(editor.editable);
+        .filter(n => n.nodeType === Node.TEXT_NODE && n.nodeValue.length);
+    const isAlreadyFormatted = style.isInSelection(editor.editable);
```

Comment:
I don't understand the reason of this renaming.

Reply:
I didn't understand the is when I read the code. So I rename it to be more clear.

Simple as that.